### PR TITLE
[WIP] Move to new backend registration

### PR DESCRIPTION
**Pull request by DeepDiver1975**

### CODE REVIEW

File: appinfo/app.php [hunk 1]
```diff
@@ -1,16 +1,8 @@
...
-		'host' => (string)$l->t('hostname'),
-		'username' => (string)$l->t('Username'),
-		'password' => (string)$l->t('Password'),
-		'root' => '&' . $l->t('Remote subfolder'),
-		'ssl' => '!' . $l->t('Secure ftps://'),
-		'port' => '&' . $l->t('Port'),
-	],
-]);
+\OC::$server->getEventDispatcher()->addListener(
+	'OCA\\Files_External::loadAdditionalBackends', function($event) {
```
Comment (DeepDiver1975) [inline appinfo/app.php]: @xenopathic ftp is never registered

Comment (Xenopathic) [inline appinfo/app.php]: If files_external_ftp is loaded after files_external, the event dispatcher trigger will have already fired, so this listener won't be activated. You probably need to check if files_external is available yet, and if not, then go via the event dispatcher. This is a consequence of not knowing which order apps are loaded in, and not having the core bits of files_external in core.

Comment (DeepDiver1975) [inline appinfo/app.php]: Well ... i know what is happening. The question is how this was supposed to ever work .

**Comment by PVince81** [comment]
Ah, damn, you already did ?

I did the same work here https://github.com/owncloud/files_external_ftp/pull/9 and it does work, so maybe use my PR instead ?

**Pull request closed without merging**